angular.json

```diff
@@ -115,5 +115,8 @@
       }
     }
   },
-  "defaultProject": "teammates"
+  "defaultProject": "teammates",
+  "cli": {
```

Comment:
This is probably unintended. Please remove

Reply:
Fixed.